user_data/strategies/freq_data_cleaner.py

```diff
@@ -0,0 +1,40 @@
+import json
+import datetime
+import time
+import os
+import sys
+from util import BACKTEST_JSON_PATH, BACKTEST_YEAR, BACKTEST_MONTH
```

Comment:
@Bemhreth I have merged by mistake here are the two comments. Please create a new pr and fix these:
1. move freq_data_cleaner.py to /freqtrade directory
2. rename BACKTEST_JSON_PATH to BACKTEST_DOWNLOADED_JSON_DATA_FILE_PATH

Reply:
Noted